lib/Compressor.js

```diff
@@ -42,15 +43,33 @@ export class Compressor extends Transformer {
    * JSON-LD document. The created instance may only be used on a single
    * JSON-LD document at a time.
    *
-   * @param {object} options - The options to use when encoding to CBOR-LD.
+   * @param {object} options - The options to use.
    * @param {documentLoaderFunction} options.documentLoader -The document
    *   loader to use when resolving JSON-LD Context URLs.
-   * @param {Map} [options.appContextMap] - A map of JSON-LD Context URLs and
-   *   their encoded CBOR-LD values (must be values greater than 32767
-   *   (0x7FFF)).
+   * @param {Map} options.keywordsTable -A map of JSON-LD keywords
+   *  and their associated CBOR-LD integer values.
+   * @param {Map} options.stringTable - A map of string values and
+   *  their associated CBOR-LD integer values.
+   * @param {Map} options.urlSchemeTable - A map of URL schemes to
+   *  their associated CBOR-LD integer values.
+   * @param {Map} options.typedLiteralTable - A map of JSON-LD types
+   *  to maps that map values of that type to their associated CBOR-LD
+   *  integer values.
    */
-  constructor({documentLoader, appContextMap} = {}) {
-    super({documentLoader, appContextMap});
+  constructor({
+    documentLoader,
+    keywordsTable,
+    stringTable,
+    urlSchemeTable,
+    typedLiteralTable
```

Comment:
Seems this could have been: `tables: {keywords, string, urlScheme, typeLiteral}` ... but I'll file an issue to consider doing this refactor later.